### PR TITLE
Added several collections

### PR DESCRIPTION
**Pull request by AlessandroLollo**
## Summary
Added docs for the following collections:

- `prefect-metricflow`
- `prefect-transform`
- `prefect-stitch`

### CODE REVIEW

File: docs/collections/catalog/prefect-metricflow.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+collectionName: prefect-metricflow
+author: Alessandro Lollo
+authorUrl: https://github.com/AlessandroLollo
+documentation: https://alessandrolollo.github.io/prefect-metricflow/
+iconUrl: https://github.com/transform-data/metricflow/raw/main/assets/MetricFlow_logo.png
```
Comment (desertaxle) [inline docs/collections/catalog/prefect-metricflow.yaml:5]: Can you add a square image with a transparent background for MetricFlow to this repo in the same location as the other collections images?

**Review by desertaxle — CHANGES_REQUESTED** [review]
Thanks for creating these collections!

In addition to my other comment, can you update the theme of the docs for `prefect-transform` os that it matches the core docs and other Prefect Collection? Here's a PR that shows the necessary changes to update the docs theme: https://github.com/PrefectHQ/prefect-shell/pull/14.

File: docs/collections/catalog/prefect-metricflow.yaml (hunk 1 above)
Comment (AlessandroLollo) [reply docs/collections/catalog/prefect-metricflow.yaml]: Hey @desertaxle 👋 
I've updated `prefect-transform` in order to backport the new template!
Feel free to take a look [here](https://github.com/AlessandroLollo/prefect-transform)

Comment (AlessandroLollo) [reply docs/collections/catalog/prefect-metricflow.yaml]: Ouch, I think I tagged the wrong Victoria in the last commit message 😅

**Comment by desertaxle** [comment]
I tried to change the base branch for this PR to `main` and it was closed for some reason. @AlessandroLollo would you be able to reopen this PR with `main` as the base branch?

**Comment by ahuang11** [comment]
Can the repos also add a link to the docs page on the right side
image

e.g. https://prefecthq.github.io/prefect-transform/

**Comment by madkinsz** [comment]
> I tried to change the base branch for this PR to `main` and it was closed for some reason. @AlessandroLollo would you be able to reopen this PR with `main` as the base branch?

Just FYI it got auto-closed because the `orion` branch and `main` branch have different histories. The `main` branch has rewritten commit messages to fix PR links.

**Comment by AlessandroLollo** [comment]
> I tried to change the base branch for this PR to `main` and it was closed for some reason. @AlessandroLollo would you be able to reopen this PR with `main` as the base branch?

Should I delete this branch before creating a new PR?

**Pull request closed without merging**